src/core/NotePlayHandle.cpp

```diff
@@ -555,15 +557,22 @@ void NotePlayHandle::updateFrequency()
 
 
 
-void NotePlayHandle::processTimePos( const TimePos& time )
+void NotePlayHandle::processTimePos(const TimePos& time, float pvalue)
 {
-	if( detuning() && time >= songGlobalParentOffset()+pos() )
+	if (detuning() && time >= songGlobalParentOffset() + pos())
```

Comment:
Since this is a guard for the entire function, perhaps we can invert it so we don't indent the entire function?
```suggestion
	if (!detuning() || time < songGlobalParentOffset() + pos()) { return; }
```